components/operator/internal/registry/secret.go

```diff
@@ -5,14 +5,11 @@ import (
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/selection"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	ServerlessRegistryDefaultSecretName              = "serverless-registry-config-default"
-	ServerlessExternalRegistrySecretName             = "serverless-registry-config"
 	ServerlessExternalRegistryLabelRemoteRegistryKey = "serverless.kyma-project.io/remote-registry"
 	ServerlessExternalRegistryLabelRemoteRegistryVal = "config"
```

Comment:
I would suggest removing these fields as well
- ServerlessExternalRegistryLabelRemoteRegistryKey
- ServerlessExternalRegistryLabelRemoteRegistryVal
because these are unused